templates/README.md.template

```diff
@@ -0,0 +1,48 @@
+# Overloaded lemmas
+
+[![Travis][travis-shield]][travis-link]
+[![Contributing][contributing-shield]][contributing-link]
+[![Code of Conduct][conduct-shield]][conduct-link]
+[![Gitter][gitter-shield]][gitter-link]
+
+[travis-shield]: https://travis-ci.com/coq-community/PROJECT_NAME.svg?branch=master
+[travis-link]: https://travis-ci.com/coq-community/PROJECT_NAME/builds
+
+[contributing-shield]: https://img.shields.io/badge/contributions-welcome-%23f7931e.svg
+[contributing-link]: https://github.com/coq-community/manifesto/blob/master/CONTRIBUTING.md
+
+[conduct-shield]: https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-%23f15a24.svg
+[conduct-link]: https://github.com/coq-community/manifesto/blob/master/CODE_OF_CONDUCT.md
+
+[gitter-shield]: https://img.shields.io/badge/chat-on%20gitter-%23c1272d.svg
+[gitter-link]: https://gitter.im/coq-community/Lobby
+
+PROJECT_DESCRIPTION
+
+## Initial author(s)
+
+NAME(S)
+
+## Current maintainer
+
+NAME (@NICKNAME)
+
+## License
+
+LICENSE
+
+## Dependencies
```

Comment:
Isn't this a part of the build instructions? Perhaps compatible Coq versions goes in meta and the dependencies go in the build instructions? I'm not sure, there might not be a clear separation here.